acasclient/acasclient.py

```diff
@@ -1930,6 +1930,32 @@ def _validate_then_save_codetable(self, url_base, codeTable: dict) -> dict:
         resp.raise_for_status()
         return resp.json()
     
+    def get_assay_scientists(self):
+        """
+        Fetch the list of possible assay scientists for  assay loading
+        """
+        resp = self.session.get("{}/api/authors?additionalCodeType=assay&additionalCodeKind=scientist&roleName=ROLE_ACAS-USERS".format(self.url))
```

Comment:
This is the same call acas uses to fetch assay scientists. There is no dedicated "api/getAssayScientists" route